nautobot/core/views/renderers.py

```diff
@@ -249,19 +249,22 @@ def get_context(self, data, accepted_media_type, renderer_context):
                         "return_url": return_url,
                     }
                     form = form_class(initial=initial)
-                table = self.construct_table(view, pk_list=pk_list)
+                delete_all = request.POST.get("_all")
+                if not delete_all:
+                    table = self.construct_table(view, pk_list=pk_list)
             elif view.action == "bulk_create":  # 3.0 TODO: remove, replaced by ImportObjects system Job
                 form = view.get_form()
                 if request.data:
                     table = data.get("table")
             elif view.action == "bulk_update":
+                edit_all = request.POST.get("_all")
                 pk_list = getattr(view, "pk_list", [])
-                if pk_list:
+                if pk_list or edit_all:
```

Comment:
Can we delete the if statement here?

Reply:
Not sure - don't we need to account for the possibility of an empty `pk_list`?

Reply:
I see we are accounting for the scenario where no objects are selected at all. That makes sense.